python/dgl/data/fraud.py

```diff
@@ -262,10 +241,9 @@ class FraudYelpDataset(FraudDataset):
     >>> label = dataset.ndata['label']
     """
 
-    def __init__(self, raw_dir=None, random_seed=2, train_size=0.7, val_size=0.1):
+    def __init__(self, raw_dir=None, train_size=0.7, val_size=0.1):
```

Comment:
Do we not want `random_seed` now?

Reply:
@zhjwy9343 please discuss or justify before resolve the conversation.

Reply:
Modified based on your comments in the latest commit.